app/models/podcast_appearance.rb

```diff
@@ -0,0 +1,11 @@
+class PodcastAppearance < ApplicationRecord
+  belongs_to :user
+  belongs_to :podcast_episode
+  validates :podcast_episode_id, uniqueness: { scope: :user_id }
+  validates :podcast_episode_id, :user_id, :role, presence: true
+  validates :role, inclusion: { in: %w[owner host guest], message: "%<value> is not a valid role" }
```

Comment:
is `owner` necessary if https://github.com/MLH-Fellowship/dev.to/issues/81 and https://github.com/forem/forem/pull/11410 are to be merged?

I guess we can find the list of a podcast owners through that if I'm not mistaken about the intent here